### PR TITLE
[ISSUE #713]  Add an logstash-output-rocketmq plugin

### PR DESCRIPTION
**Pull request by hehejiejie**
## What is the purpose of the change

Add an logstash-output-rocketmq plugin,named rocketmq-logstash.

## Brief changelog

It is an external which enables logstash to output events to rocketmq.

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: rocketmq-logstash/CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+The following is a list of people who have contributed ideas, code, bug
+reports, or in general have helped logstash along its way.
+
+Contributors:
+* Aaron Mildenstein (untergeek)
```
Comment (vongosling) [inline rocketmq-logstash/CONTRIBUTORS:5]: Does it contribute to this plugin?

**Review by vongosling — CHANGES_REQUESTED** [review]

**Comment by vongosling** [comment]
Do not import some dependencies directly in bin directory.

File: rocketmq-logstash/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## 2.0.3
+  - Docs: Set the default_codec doc attribute.
```
[the remark this answers is not in the document]
Comment (hehejiejie) [reply rocketmq-logstash/CHANGELOG.md:2]: This project is initialized from an example in [https://github.com/logstash-plugins/logstash-output-example.git](url), so there are some default files I didn't modify.

[the remark this answers is not in the document]
Comment (vongosling) [reply rocketmq-logstash/CHANGELOG.md:2]: You could change it before commit to here.

File: rocketmq-logstash/CONTRIBUTORS (hunk 1 above)
Comment (hehejiejie) [reply rocketmq-logstash/CONTRIBUTORS:5]: no...

**Comment by vongosling** [comment]
I will close this pr. and glad to see redacting for the codebase. Pls remember, do not import other code without any declaration here.

**Pull request closed without merging**